Extension/src/LanguageServer/client.ts

```diff
@@ -2614,6 +2639,40 @@ export class DefaultClient implements Client {
         this.model.codeAnalysisTotal.Value = total;
     }
 
+    private async insertDoxygenComment(result: GenerateDoxygenCommentResult): Promise<void> {
+        const editor: vscode.TextEditor | undefined = vscode.window.activeTextEditor;
+        if (editor) {
+            const oldVersion: number = result.fileVersion;
+            const newVersion: number | undefined = openFileVersions.get(editor.document.uri.toString());
+
+            if (newVersion !== undefined && oldVersion !== undefined && oldVersion === newVersion &&
+                result?.initPosition.line === editor.selection.active.line && result?.initPosition.character === editor.selection.active.character) {
+                if (result?.contents) {
+
+                    if (result.contents.length > 1) {
+                        const workspaceEdit: vscode.WorkspaceEdit = new vscode.WorkspaceEdit();
+                        const edits: vscode.TextEdit[] = [];
+
+                        if (vscode.window.activeTextEditor) {
```

Comment:
It seems like this check condition and the 3 previous "if"s can be combined into 1 single if that returns.